blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricsIndexHandler.java

```diff
@@ -24,7 +23,7 @@
     private DiscoveryIO discoveryHandle;
 
     public HttpMetricsIndexHandler() {
-        loadDiscoveryModule();
+        discoveryHandle = QueryDiscoveryModuleLoader.getDiscoveryInstance();
```

Comment:
@chinmay-gupte what do you think about not making discoveryHandle a static variable? It makes testing very hard. We could fake dependency injection this way and pass mocked out DiscoveryIO to the HttpMetricsIndexHandler

```
public HttpMetricsIndexHandler(DiscoveryIO discoveryHandler) {
}
```